app/models/repo.js

```diff
@@ -136,17 +140,27 @@ const Repo = VcsEntity.extend({
     return this._buildObservableArray(builds);
   }),
 
-  pullRequests: computed('id', function () {
-    let id = this.id;
-    const builds = this.store.filter('build', {
-      event_type: 'pull_request',
-      repository_id: id,
-    }, (b) => {
-      const isPullRequest = b.get('eventType') === 'pull_request';
-      return this._buildRepoMatches(b, id) && isPullRequest;
-    });
-    return this._buildObservableArray(builds);
-  }),
+  // Using dynamicQuery effectively relies on accurate pagination, which
+  // is not possible if skip_count=true, which is the default for queries in
+  // the build adapter. force_count allows overriding this behavior, but we
+  // should use it cautiously for performance reasons. I think it may
+  // be fine for pull requests though?
```

Comment:
Pagination is used in the load more component to show/hide the "Show More" button, as well as the missing notice. If `skip_count=true`, then the Show More buttons is always visible, and the missing notice never is. Which is not ideal, but better than the Missing Notice showing when it should not, which is the current situation. Still, I might make some updates today to allow these things to work properly even if `skip_count=true`

Reply:
Ok I made some updates to dynamicQuery to allow handling pagination when skip_count = true. If needed, we can move that logic elsewhere (like into the load more component, or possibly even into the api?).